owncloudApp/src/main/java/com/owncloud/android/presentation/transfers/TransferListComposeFragment.kt

```diff
@@ -0,0 +1,532 @@
+                    color = colorResource(id = R.color.color_accent),
+                    text = requireContext().getString(headerTitleStringRes(status)).uppercase()
+                )
+                Spacer(modifier = Modifier.weight(1f))
+                Text(
+                    modifier = Modifier
+                        .layoutId("textViewFileCount")
+                        .padding(end = 16.dp),
+                    fontWeight = FontWeight.Bold,
+                    color = colorResource(id = R.color.half_black),
+                    text = fileCountText.uppercase()
+                )
+            }
+            Divider(
+                modifier = Modifier
+                    .fillMaxWidth(),
+                color = colorResource(id = R.color.filelist_icon_background),
+                thickness = 2.dp
+            )
+            Row {
+                if (status == TransferStatus.TRANSFER_FAILED || status == TransferStatus.TRANSFER_SUCCEEDED) {
+                    Button(
+                        modifier = Modifier
+                            .layoutId("uploadListGroupButtonClear")
+                            .padding(start = 13.dp),
+                        onClick = {
+                            if (status == TransferStatus.TRANSFER_FAILED) {
+                                transfersViewModel.clearFailedTransfers()
+                            } else {
+                                transfersViewModel.clearSuccessfulTransfers()
+                            }
+                        },
+                        shape = RoundedCornerShape(1.dp),
+                        colors = ButtonDefaults.buttonColors(backgroundColor = colorResource(id = R.color.primary_button_background_color))
+                    ) {
+                        Text(
+                            text = stringResource(id = R.string.action_upload_clear).uppercase(),
+                            color = colorResource(id = R.color.white),
+                        )
+                    }
+                }
+                if (status == TransferStatus.TRANSFER_FAILED) {
+                    Button(
+                        modifier = Modifier
+                            .layoutId("uploadListGroupButtonRetry")
+                            .padding(start = 8.dp),
+                        onClick = { transfersViewModel.retryFailedTransfers() },
+                        shape = RoundedCornerShape(1.dp),
+                        colors = ButtonDefaults.buttonColors(backgroundColor = colorResource(id = R.color.color_accent))
+                    ) {
+                        Text(
+                            text = stringResource(id = R.string.action_upload_retry).uppercase(),
+                            color = colorResource(id = R.color.white),
+                        )
+                    }
+                }
+
+            }
+        }
+        if (status == TransferStatus.TRANSFER_SUCCEEDED || status == TransferStatus.TRANSFER_FAILED){
```

Comment:
To keep consistency with line 427, I would express the condition the other way around (first checking if status is failed)

Reply:
Changed 🚀